test/extended/coreos/coreos.go

```diff
@@ -28,7 +28,8 @@ var _ = g.Describe("[sig-coreos] [Conformance] CoreOS bootimages", func() {
 	defer g.GinkgoRecover()
 
 	oc := exutil.NewCLIWithoutNamespace("coreos")
-	g.It("TestBootimagesPresent", func() {
+
+	g.It("TestBootimagesPresent [apigroup:machineconfiguration.openshift.io]", func() {
```

Comment:
Checking https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/27400/pull-ci-openshift-origin-master-e2e-aws-ovn-cgroupsv2/1573331257285152768 the test is still running

Reply:
This comes from the CRD that MCO installs. If its not there then there is certainly no MCO.